### PR TITLE
o Add SWnet def file

### PR DESCRIPTION
**Pull request by rajeeja**
Please test

### CODE REVIEW

**Review by wilke — CHANGES_REQUESTED** [review]
Please add 

%environment
    export IMPROVE_MODEL_DIR=/usr/local/{{MODEL}}


This will be used in train.sh, see https://github.com/JDACS4C-IMPROVE/Singularity/blob/develop/src/templates/train.sh

**Comment by rajeeja** [comment]
#6 this might fix it.

File: definitions/SWnet.def [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+%labels
+	MANTAINER Gihan Panapitiya
+
+%setup
+	cp ./src/Singularity_gpu_fix.sh $SINGULARITY_ROOTFS
+	# add local url of this repository for testing
+        # cp -r /path/to/local/SWnet $SINGULARITY_ROOTFS/usr/local/
+
+%environment
+	export IMPROVE_MODEL_DIR=/usr/local/{{MODEL}}
```
Comment (wilke) [inline definitions/SWnet.def:13]: This is a template. Replace {{MODEL}} with name of the model dir.

**Review by wilke — COMMENTED** [review]
Apart from the environment section which is not functional, I don't see a difference to the version in the switch-to-improve branch.

**Pull request closed without merging**